### PR TITLE
feat(interface): Added topic interface with recognitions

### PR DESCRIPTION
**Pull request by reinzor**

### CODE REVIEW

**Comment by Timple** [comment]
Code buildup seems ok. 

Is this tested?

Lets fix travis first

**Comment by reinzor** [comment]
It's tested. Fixing travis is a good idea :)

-Rein

File: image_recognition_util/src/image_recognition_util/image_writer.py [hunk 1]
```diff
@@ -62,4 +85,23 @@ def write_raw(dir_path, image):
...
\ No newline at end of file
+    return True
+
+
+def get_annotated_cv_image(cv_image, recognitions):
+    """
+    Gets an annotated CV image based on recognitions, drawin using cv.rectangle
+    :param cv_image: Original cv image
+    :param recognitions: List of recognitions
+    :return: Annotaed image
```
Comment (LoyVanBeek) [inline image_recognition_util/src/image_recognition_util/image_writer.py]: typo

File: image_recognition_util/src/image_recognition_util/image_writer.py [hunk 2]
```diff
@@ -1,8 +1,31 @@
+import numpy as np
+
 import cv2
 import os
 import datetime
 
 
+def color_map(N=256, normalized=False):
```
Comment (LoyVanBeek) [inline image_recognition_util/src/image_recognition_util/image_writer.py]: Missing docstring. Suggestion:
```python
'''Generate an RGB color map of N different colors
:param N : int amount of colors to generate
:param normalized: bool indicating range of each channel: float32 in [0, 1] or int in [0, 255]
:return a numpy.array of shape (N, 3) with a row for each color and each row is [R,G,B]
'''
```

File: openface_ros/scripts/face_recognition_node [hunk 3]
```diff
@@ -4,40 +4,62 @@ import rospy
...
 class OpenfaceROS:
-    def __init__(self, align_path, net_path, save_images_folder):
+    def __init__(self, align_path, net_path, save_images_folder, topic_save_images,
+                 service_save_images, topic_publish_result_image, service_publish_result_image):
         """
         OpenfaceROS class that wraps the FaceRecognizer in a ROS node
         :param align_path: dlib face align path
         :param net_path: openface neural net path
         :param save_images_folder: path where to store the images
+        :param topic_save_images: whether to save images originated from image topic callback
```
Comment (LoyVanBeek) [inline openface_ros/scripts/face_recognition_node]: Nitpicking: I'd like to see the type of each param, e.g, I think ```bool``` in these cases

File: image_recognition_util/src/image_recognition_util/image_writer.py (hunk 2 above)
Comment (reinzor) [reply image_recognition_util/src/image_recognition_util/image_writer.py]: Processed

File: openface_ros/scripts/face_recognition_node (hunk 3 above)
Comment (reinzor) [reply openface_ros/scripts/face_recognition_node]: Description is clear enough I.M.O.

**Pull request merged**